cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/node/listener/PlayerManagerListener.java

```diff
@@ -62,15 +62,15 @@ public void handle(ChannelMessageReceiveEvent event) {
             case "update_offline_cloud_player": {
                 ICloudOfflinePlayer cloudOfflinePlayer = event.getData().get("offlineCloudPlayer", CloudOfflinePlayer.TYPE);
 
-                if (cloudOfflinePlayer != null) {
+                if (cloudOfflinePlayer != null) { //todo shouldn't we call the BridgeUpdateCloudOfflinePlayerEvent here? and if this message comes from a service (and not from a node), call updateOfflinePlayer instead of updateOfflinePlayer0
                     NodePlayerManager.getInstance().updateOfflinePlayer0(cloudOfflinePlayer);
                 }
             }
             break;
             case "update_online_cloud_player": {
                 ICloudPlayer cloudPlayer = event.getData().get("cloudPlayer", CloudPlayer.TYPE);
 
-                if (cloudPlayer != null) {
+                if (cloudPlayer != null) { //todo shouldn't we call the BridgeUpdateCloudPlayerEvent here? and if this message comes from a service (and not from a node), call updateOnlinePlayer instead of updateOnlinePlayer0
```

Comment:
The event gets called in the BridgeCustomChannelMessageListener

Reply:
Removed